test/e2e_node/container_lifecycle_test.go

```diff
@@ -245,14 +245,15 @@ var _ = SIGDescribe(framework.WithNodeConformance(), "Containers Lifecycle", fun
 							// to complete.
 							// Note that we've observed approximately a 2s
 							// delay before the postStart hook is called.
-							// 10s > 1s + 2s(estimated maximum delay) + other possible delays
+							// 10s > 1s + 1s + 2s(estimated maximum delay) + other possible delays
 							Delay:    10,
 							ExitCode: 0,
 						}),
 						Lifecycle: &v1.Lifecycle{
 							PostStart: &v1.LifecycleHandler{
 								Exec: &v1.ExecAction{
 									Command: ExecCommand(prefixedName(PostStartPrefix, regular1), execCommand{
+										StartDelay:    1,
```

Comment:
Just wondering if this test case should even check if poststart and regular container run together, especially if this requires artificially delaying poststart.

Reply:
I think it's still worth checking this condition in `RunTogether()`:

https://github.com/kubernetes/kubernetes/blob/03ad8e5b04c5cdeefc6ff97f68a549e210ce44cb/test/e2e_node/container_lifecycle_pod_construction.go#L139-L141

Although this doesn't look the primary purpose of this test case, I think it would be better to confirm that a regular container and a postStart hook run concurrently somewhere.